src/components/Provider/Provider.tsx

```diff
@@ -36,22 +36,26 @@ class Provider extends Component<any, any> {
 
   static Consumer = ProviderConsumer
 
-  renderStaticStyles = felaRenderer => {
+  renderStaticStyles = () => {
+    // RTL WARNING
```

Comment:
maybe it would be a good idea to include reference on docs where it is stated that
> no further processing with plugins will be done for static styles

http://fela.js.org/docs/advanced/StaticStyle.html

Reply:
Thanks, link added.